app/src/main/java/ch/epfl/sweng/studdybuddy/MainActivity.java

```diff
@@ -1,48 +1,63 @@
 package ch.epfl.sweng.studdybuddy;
 
 import android.content.Intent;
-import android.graphics.Bitmap;
-import android.provider.MediaStore;
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
 import android.view.View;
 import android.widget.Button;
-import android.widget.ImageView;
+
+import com.google.android.gms.auth.api.signin.GoogleSignInClient;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
 
 public class MainActivity extends AppCompatActivity
 {
+    private AuthManager mAuth = null;
```

Comment:
Why not static? why not instantiated in onCreate?(just asking...)

Reply:
I need to modify the AuthManager for the tests. So I can't instantiate it in onCreate(). 
I don't see why making the authmanager static would allow modify it in a test between the call of the activity's constructor and of its onCreate() method

Reply:
Ok sure, makes sense, I was thinking about something else. Thank you